doc/pod-creation-example.md

```diff
@@ -0,0 +1,81 @@
+# Example - Allocating MAC addresses to a Pod
+
+## Create a a bridge on the node:
+
+Using ovs-vsctl, we will add a bridge br1 on node01
+
+```bash
+node=node01
+./cluster/cli.sh ssh ${node} -- sudo yum install -y http://cbs.centos.org/kojifiles/packages/openvswitch/2.9.2/1.el7/x86_64/openvswitch-2.9.2-1.el7.x86_64.rpm http://cbs.centos.org/kojifiles/packages/openvswitch/2.9.2/1.el7/x86_64/openvswitch-devel-2.9.2-1.el7.x86_64.rpm http://cbs.centos.org/kojifiles/packages/dpdk/17.11/3.el7/x86_64/dpdk-17.11-3.el7.x86_64.rpm
+./cluster/cli.sh ssh ${node} -- sudo systemctl daemon-reload
+./cluster/cli.sh ssh ${node} -- sudo systemctl restart openvswitch
+
+./cluster/cli.sh ssh ${node} -- sudo ovs-vsctl add-br br1
+```
+
+## Create a NetworkAttachmentDefinition:
+
+This example will use [ovs-cni](https://github.com/kubevirt/ovs-cni/).
+
+```bash
```

Comment:
Use yaml highlighting

Reply:
but it's bash.. how will formatting it as yaml make it better?

Reply:
Done